app/javascript/controllers/step_by_step_nav_controller.js

```diff
@@ -1,4 +1,4 @@
-require('../src/javascripts/step-by-step-nav.js');
```

Comment:
Does this break the green lanes wizard?

Reply:
Given this now imported in the map, the require isn't needed.